recipes/msgpack-cxx/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.36.0"
+
+class MsgpackCXXConan(ConanFile):
+    name = "msgpack-cxx"
+    description = "The official C++ library for MessagePack"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/msgpack/msgpack-c"
+    topics = ("msgpack", "message-pack", "serialization")
+    license = "BSL-1.0"
+    generators = "cmake"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        self.options["boost"].header_only = False
+        self.options["boost"].without_chrono = False
+        self.options["boost"].without_context = False
+        self.options["boost"].without_system = False
+        self.options["boost"].without_timer = False
```

Comment:
If this is a requirement, please raise an exception in `validate(self)` if any of these options have incorrect values (in addition of setting the values in `configure(self):`

Reply:
Fixed